datadog_checks_base/datadog_checks/base/checks/openmetrics/mixins.py

```diff
@@ -506,12 +515,22 @@ def process_metric(self, metric, scraper_config, metric_transformers=None):
         # If targeted metric, store labels
         self._store_labels(metric, scraper_config)
 
-        if metric.name in scraper_config['ignore_metrics']:
+        if metric.name in scraper_config['_ignored_metrics']:
             self._send_telemetry_counter(
                 self.TELEMETRY_COUNTER_METRICS_IGNORE_COUNT, len(metric.samples), scraper_config
             )
             return  # Ignore the metric
 
+        for ignore_pattern in scraper_config['_ignored_patterns']:
+            if fnmatchcase(metric.name, ignore_pattern):
```

Comment:
This will cause a large slow down. No way around it I guess, just saying...

Reply:
is it worth adding a `if config['ignore_metrics']` or the `for` on the empty set if fast enough?

Reply:
here is a small benchmark - it's almost the same
```python
>>> timeit('for x in s:    print("found")', setup='s = set(); x = "randomstring"',number=100000)
0.004889011383056641
>>>
>>> timeit('if map:    print("found")', setup='map = dict()', number=100000)
0.003404855728149414

Reply:
in absolute it's fast yes, but looks like we could use the 30% speedup 👍

Reply:
here is a follow-up PR https://github.com/DataDog/integrations-core/pull/5764